docs/xostor/xostor.md

```diff
@@ -852,3 +852,64 @@ Note: iptables config must also be modified to remove LINSTOR port rules (edit `
 For more info:
 - https://linbit.com/blog/linstors-auto-evict/
 - https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-linstor-auto-evict
+
+### How to-enable dm-cache?
+
+:::warning
+This feature is currently experimental and is not covered by XCP-ng support.
```

Comment:
"not covered by Vates professional support" (we try to progressively standardize all our products under the Vates brand). You can even link to  https://vates.tech/pricing-and-support for "Vates Pro Support" text.